alf/drivers/policy_driver.py

```diff
@@ -73,9 +73,21 @@ def __init__(self,
         standard_metrics = [
             tf_metrics.NumberOfEpisodes(),
             tf_metrics.EnvironmentSteps(),
-            tf_metrics.AverageReturnMetric(buffer_size=metric_buf_size),
-            tf_metrics.AverageEpisodeLengthMetric(buffer_size=metric_buf_size),
         ]
+        # This is a HACK.
+        # Due to tensorflow metric API change:
+        # https://github.com/tensorflow/agents/commit/b08a142edf180325b63441ec1b71119c393c4a64,
+        # after tensorflow v20190807, these two metrics will cause error during
+        # playing of the trained model, when num_parallel_envs > 1.
```

Comment:
For playing with the trained model, should the batch_size be always one?

Reply:
OK, I thought num_parallel_envs > 1 is for playing. What you really mean is that if during training we have num_parallel_envs > 1 and we have num_parallel_envs==1 for playing, after this update, then these two metrics will not work.